docs/01-Introduction.md

```diff
@@ -1,50 +1,55 @@
 # 1.0 - Introduction
 
 ### 1.1 What is LORIS-MRI? 
-LORIS-MRI is the backbone of the imaging component that makes up LORIS.
- It is maintained in a separate repository so that it can be installed
- on the file server and separated from the web server.
+
+LORIS-MRI is the backbone of the imaging component that makes up LORIS. 
+It is maintained in a separate repository so that it can be installed 
+on the file server and separated from the web server.
 These documents assume you have some knowledge regarding LORIS and
 a functioning installation. For information regarding LORIS itself,
 please consult the [LORIS wiki][1].
 
 LORIS-MRI is a set of libraries responsible for the insertion,
-organization, and archival of uploaded imaging datasets.
- It expects an uploaded, compressed file containing
-a [DICOM][2] study composed of several DICOM files. These DICOM files
- will be archived on the server and converted to [MINC][3] and (optionally)
+organization, and archival of uploaded imaging datasets. 
```

Comment:
archival -> archiving 
(archival is much more common as an adjective so the sentences reads a tiny bit funny)